apps/cookbook/src/app/examples/example-configuration-wrapper/example-configuration-wrapper.component.scss

```diff
@@ -28,20 +28,24 @@ $button-z-index: $card-z-index + 1;
     }
 
     kirby-card {
+      height: 500px;
+      overflow-y: auto;
       opacity: 0;
       visibility: hidden;
       rotate: 45deg;
-      transition-property: opacity, rotate;
+      transition-property: opacity, rotate, scale;
       transition-duration: utils.get-transition-duration('quick');
       transition-timing-function: utils.get-transition-easing('enter-exit');
       transform-origin: top left;
+      scale: 0.5;
```

Comment:
This scaling is introduced to fix scrollbar flicker when the example is shown in a device-frame. When rotated 45deg, the diagonal example config height becomes larger than the screen height, causing a scrollbar to flicker briefly until the example config has animated fully

Reply:
<img width="387" alt="image" src="https://github.com/kirbydesign/designsystem/assets/42470636/1bb0c2cf-c2e3-42aa-8cce-796b5d0f41f4">